docs/resources/built-in-rules.md

```diff
@@ -328,3 +328,225 @@ Here is the equivalent of the `recommended` configuration values:
 
 OpenAPI-cli supports [custom rules](./custom-rules.md).
 However, if you have an idea for a built-in rule you believe will benefit the greater API community, please [open an issue](https://github.com/Redocly/openapi-cli/issues/new).
+
+## Generic built-in rules
+
+You can use special `enforcements` rule for cases which our other built-in rules do not cover.
+There are [a set of generic built-in rules](#list-of-generic-built-in-rules) which can be configured under `enforcements` to enforce different generic usecases.
```

Comment:
Won't it be confusing to have rules inside a rule? How about calling them `enforcements`? @skadinna @adamaltman

Reply:
I think assertions is best